airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -0,0 +1,140 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/file",
+
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "File Source Spec",
+    "type": "object",
+    "required": ["url", "storage"],
+
+    "properties": {
+      "format": {
+        "type": "string",
+        "enum": [
+          "csv",
+          "json",
+          "html",
+          "excel",
+          "feather",
+          "parquet",
+          "orc",
+          "pickle"
+        ],
+        "default": "csv"
+      },
+      "reader_options": {
```

Comment:
What is this?

You can add documentation for these properties

Reply:
Yes i'm missing a lot of documentation, I just had enough to ship the integration so far. 
Will detail more writings about it now